Saasu.API.Client/Saasu.API.Client.nuspec

```diff
@@ -3,9 +3,9 @@
 	<metadata>
 		<id>Saasu.API.Dotnet.Sdk</id>
 		<title>API SDK for the Saasu API</title>
-		<version>1.1.7</version>
+		<version>1.1.7.2</version>
```

Comment:
Please don't change the nuspec file.  We will do this for a release.

Reply:
This commit was not intended to be a part of the pull request.  Because there was no response to our request, we made this change internally so we could push to our private nuget repository without conflicts.